pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -1452,7 +1452,7 @@ class Timestamp(_Timestamp):
         Examples
         --------
         >>> pd.Timestamp.fromtimestamp(1584199972)
-        Timestamp('2020-03-14 15:32:52')
+        Timestamp('2020-03-14 16:32:52')
```

Comment:
the exact output here depends on your tz's local time
```python
In [1]: pd.Timestamp.fromtimestamp(1584199972)
Out[1]: Timestamp('2020-03-14 15:32:52')

In [2]: import os

In [3]: os.environ['TZ'] = 'Europe/Brussels'

In [4]: import time

In [5]: time.tzset()

In [6]: pd.Timestamp.fromtimestamp(1584199972)
Out[6]: Timestamp('2020-03-14 16:32:52')
```

I think this can probably be just removed directly from `ci/code_checks.sh` (I presume CI runs in UTC?), or could just doctest skip it

Reply:
Thank you, it was my mistake. I had to read the next line `1457`.
I guess CI runs in UTC, because I get a one-hour difference running validation locally.

Can we leave `pandas.Timestamp.fromtimestamp` in `ci/code_checks.sh` to ignore this function while validating docstrings and use the original example for UTC?

Reply:
ideally we shoudn't have the `excludes` list in `ci/code_checks.sh`, the objective is to remove that - I think if you add `# doctest skip` (something like that, check other docstrings for the exact syntax) just to this doctest, and keep it removed from `ci/code_checks`, that should be enough

Reply:
Thank you. I did as you suggested and updated the PR.